sansio_lsp_client/client.py

```diff
@@ -87,6 +190,11 @@ def __init__(
     def state(self) -> ClientState:
         return self._state
 
+    @property
+    def is_initialized(self):
+        return (self._state != ClientState.NOT_INITIALIZED
+                    and self._state != ClientState.WAITING_FOR_INITIALIZED)
```

Comment:
Do we really need this?

Reply:
Seems much nicer then dealing with assertion errors everywhere